dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -35,8 +35,4 @@
     %textarea#level_markdown_textarea
     #level_markdown_textarea_preview
 
-  .markdown.teacher
-    %p Edit Teacher Only Markdown:
-
-    %textarea#level_teacher_markdown_textarea
-    #level_teacher_markdown_textarea_preview
+  = render partial: 'levels/editors/teacher_only_markdown', locals: {f: f}
```

Comment:
I left this here because of the code on line 28 which determines if a dsl uses markdown.